cli/src/index.ts

```diff
@@ -14,4 +15,14 @@ program
     .description('Produces an SVG file representing a visualization of the CALM Specification.')
     .action(visualize);
 
+program
+    .command('generate')
+    .description('Generate an instantiation from a CALM pattern file.')
+    .requiredOption('-s, --source <source>', 'Path to the pattern file to use. May be a file path or a URL.')
```

Comment:
For visualize we use -i --input as the flag, should we use the same for consistency?

Reply:
Happy to make this change - we should try to keep this flag consistent across all 3 commands.
For validation we have pattern, instantiation and meta-schema which complicates things - would perhaps -p --pattern make more sense for all 3 to use as this would help with consistency?

Reply:
I'm good with that, I agree that consistency is the important thing rather than what the flag actually is.